### DISCUSSION TITLE
Ability to disable flyout animation

### DISCUSSION THREAD

**Discussion by notsyncing12309** [File-New-Project/EarTrumpet#1362] [Ideas]
Some users experience opening animation lags. As I understand it, this problem cannot be solved just like that, so it would be nice to be able to disable animations or set their speed.
I know it's possible to turn off all animations through the Windows settings, but if the animation is lagging only in a specific application, then it's better to be able to turn it off only in that application.

**Comment by riverar** [comment]
We recently improved the flyout situation (https://github.com/File-New-Project/EarTrumpet/pull/1171) so it's not clear if this is needed. Have you tried it out in dev builds?
